src/probnum/quad/_bayesquad.py

```diff
@@ -98,14 +103,14 @@ def bayesquad(
     Examples
     --------
     >>> import numpy as np
-    >>> np.random.seed(0)
+
     >>> input_dim = 1
     >>> domain = (0, 1)
     >>> def f(x):
     ...     return x
     >>> F, info = bayesquad(fun=f, input_dim=input_dim, domain=domain)
     >>> print(F.mean)
-    0.5000
+    0.5
```

Comment:
the seed does not have any effect anymore. To recreate the other functionality, I could do
```python
rng = np.random.default_rng(seed=0)
F, info = bayesquad(fun=f, input_dim=input_dim, domain=domain, rng=rng)
```
or even
```python
rng = np.random.default_rng(seed=0)
F, info = bayesquad(fun=f, input_dim=input_dim, domain=domain, policy="bmc", rng=rng)
```
Since I thought the simplest interface would be the target here, I decided against that. But let me know if you want me to change it.

Reply:
> I don't really get what you mean by
setting `np.random.seed(1)` has no effect since all the underlying sampling uses fixed random number generators

> other functionality
Fixing a seed

Reply:
Then this is fine :)